### DISCUSSION TITLE
Input box in flixel-ui

### DISCUSSION THREAD

**Discussion by bgm145632** [HaxeFlixel/flixel#2463] [General]
Can I input other languages in the input box?

**Comment by Geokureli** [comment]
for special characters like accented chars, it likely depends on the font you use. try a system font

**Reply by bgm145632** [reply]
How to use it?

**Comment by RichardBray** [comment]
I've never used flixel-ui before. Out of curiosity what are you trying to achieve? Maybe the built in FlxText might be more appropriate.
